squirrel_interaction/src/squirrel_interaction/board/serial_api.py

```diff
@@ -5,13 +5,13 @@
 
 The board controls the following devices:
 
-    - Head Motor (pan) (-180º ~ 180ª)
+    - Head Motor (pan) (-180º ~ 180ª) - head_controller
```

Comment:
It is only a comment but can you please also change the wrong letters ª to ° while we are already changing this file.
I think this happened 3 times. Thank you.